tests/lib/autofix.js

```diff
@@ -172,3 +172,46 @@ describe('Complex autofix test cases', () => {
     })
   })
 })
+
+describe('Autofix test cases with Typescript', () => {
```

Comment:
Could you change it to test case using rule tester?
https://github.com/vuejs/eslint-plugin-vue/blob/master/tests/lib/rules/order-in-components.js

Reply:
done!